### ISSUE TITLE
TAG: Strategy vs coordination

### ISSUE THREAD

**Issue by dontcallmedom** [darobin/w3c-structure#5]
The document hints at a potential dual role for the TAG (or at a potential split of things people would perhaps associated with the TAG today).

I think that duality ought to be discussed in more depth (because I agree it's an important one).

The role of the technical Team also relates to these two roles, which may inspire more discussions about the articulation of these roles and these groups.

**Comment by darobin** [comment]
I agree that it should be discussed more, but I would like to have the discussion about it amongst interested folks before putting something in the doc? If you have strong/clear views on exactly how to make this work well, I'd be happy to incorporate as an opening position for debate.
